### PR TITLE
updated @types/react-image-crop to match the newest version of react-image-crop

### PR DESCRIPTION
**Pull request by Englund0110**
updated tests

added missing semicolon

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/DominicTobias/react-image-crop/releases/tag/7.0.0
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-image-crop/index.d.ts [hunk 1]
```diff
@@ -67,6 +60,7 @@ declare class ReactCrop extends Component<ReactCrop.ReactCropProps> {
         width: number,
         height: number,
     };
+    resolveCropAndTriggerChange: (crop: ReactCrop.Crop, image: HTMLImageElement) => ReactCrop.Crop;
```
Comment (PranavSenthilnathan) [inline types/react-image-crop/index.d.ts]: I think this should just be `resolveCrop`:
https://github.com/DominicTobias/react-image-crop/blob/master/lib/ReactCrop.js#L101

**Review by PranavSenthilnathan — CHANGES_REQUESTED** [review]

**Comment by Englund0110** [comment]
@PranavSenthilnathan Hi, thanks for the reviewing. I have made the changes that you suggested. Please take a look. :-)

File: types/react-image-crop/index.d.ts [hunk 2]
```diff
@@ -12,31 +13,24 @@ export as namespace ReactCrop;
...
-        x: number;
-        y: number;
-        width: number;
-        height: number;
-    }
-
     interface ReactCropProps {
         src: string;
-        crop?: Crop;
+        crop: Crop;
```
[the remark this answers is not in the document]
Comment (Englund0110) [reply types/react-image-crop/index.d.ts]: I reverted it, so it stays optional.

File: types/react-image-crop/index.d.ts (hunk 1 above)
Comment (Englund0110) [reply types/react-image-crop/index.d.ts]: Of course!

**Comment by Englund0110** [comment]
Can anyone have a look at the changes?

**Review by PranavSenthilnathan — APPROVED** [review]

**Comment by PranavSenthilnathan** [comment]
Congratulations on your first DefinitelyTyped contribution! Thank you for being a part of the community!

**Pull request merged**